package.json

```diff
@@ -8,6 +8,7 @@
     "chalk-animation": "^1.6.0",
     "cross-env": "^7.0.3",
     "dotenv": "^16.0.0",
+    "flow": "^0.2.3",
```

Comment:
I don't see this dependency being used anywhere. Did you mean to add it?

Reply:
removed